deploy/script/katello-debug.sh

```diff
@@ -5,8 +13,23 @@ then
   exit 1
 fi
 
-# General stuff
-add_files "/var/log/audit/audit.log"
+# Installer
+add_files "/var/log/katello-installer/*"
+add_files "/etc/katello-installer/*"
```

Comment:
Due to the installer break out, {/etc,/var/log}/capsule-installer and {/etc,/var/log}/sam-installer as well